src/ServiceControlInstaller.PowerShell/ServiceControlInstaller.Powershell.dll-help.xml

```diff
@@ -833,77 +921,78 @@ Get-SecurityIdentifier MyDomain\Foo</dev:code>
 			<command:parameter required="true" variableLength="false" globbing="false" pipelineInput="false" position="named">
 				<maml:name>Name</maml:name>
 				<maml:description>
-					<maml:para>The name of the ServiceControl Instance</maml:para>
+					<maml:para>The name of the ServiceControl Instance.  This will be used as the Windows Service Name.  The name must be unique for all Windows services and not contain any spaces or special characters.</maml:para>
 				</maml:description>
 				<command:parameterValue required="true" variableLength="false">String</command:parameterValue>
 			</command:parameter>
 			<command:parameter required="true" variableLength="false" globbing="false" pipelineInput="false" position="named">
 				<maml:name>InstallPath</maml:name>
 				<maml:description>
-					<maml:para>The directory to use for this ServiceControl Instance.  The binary files are copied here.</maml:para>
+					<maml:para>The directory to use for this ServiceControl Instance.  The binary files are copied here.  This should be unique for each instance of ServiceControl and should not be a subfolder of the DBPath or LogPath</maml:para>
 				</maml:description>
 				<command:parameterValue required="true" variableLength="false">String</command:parameterValue>
 			</command:parameter>
 			<command:parameter required="true" variableLength="false" globbing="false" pipelineInput="false" position="named">
 				<maml:name>DBPath</maml:name>
 				<maml:description>
-					<maml:para>The directory that will contain the RavenDB database for this ServiceControl Instance</maml:para>
+					<maml:para>The directory that will contain the RavenDB database for this ServiceControl Instance, This should be unique for each instance of ServiceControl and should not be a subfolder of the InstallPath or LogPath</maml:para>
```

Comment:
Should we mention RavenDB explicitly or call it the "embedded database files" or something?

Should the comma be a period?